apps/mymedicare_cb/authorization.py

```diff
@@ -15,6 +15,11 @@
 logger = logging.getLogger('hhs_server.%s' % __name__)
 
 
+class BBMyMedicareSLSxSignoutException(APIException):
+    # BB2-544 custom exception
+    status_code = status.HTTP_502_BAD_GATEWAY
```

Comment:
> What do you think about creating some custom, easy to read messages for these custom exceptions? Something plain english, so when looking through the logs its very easy to parse out what happened?

These custom exceptions end up showing to the beneficiary with the message contained in the settings.MEDICARE_ERROR_MSG text. It has info about contacting medicare.gov, etc..   A possible improvement might be to move that message in to the classes instead of when they are raised?  I think this is OK for now and maybe something for the backlog.